qa/rpc-tests/wallet_mergetoaddress_2.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import shutil
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.authproxy import JSONRPCException
+from test_framework.util import assert_equal, initialize_chain_clean, \
+    start_node, connect_nodes_bi, sync_blocks, sync_mempools, \
+    wait_and_assert_operationid_status
+
+import time
+from decimal import Decimal
```

Comment:
There are a few unused imports that can be removed, in particular `JSONRPCException`, `sync_blocks, sync_mempools`, `time` and `Decimal`

Reply:
Done